cypress/utils/channelsUtils.js

```diff
@@ -0,0 +1,40 @@
+import Channels from "../apiRequests/Channels";
+
+class ChannelsUtils {
+  channels = new Channels();
+
+  deleteChannels(nameStartsWith) {
+    this.channels.getChannels().then(resp => {
+      const channelsArray = new Set(resp.body.data.channels);
+      if (channelsArray) {
```

Comment:
I think an early return could be nice here. It makes the code look a bit more simple in cases such as this, since there's less braces wrapping

```suggestion
      if (!channelsArray) { return; }
      ...rest
```